nd4j/nd4j-backends/nd4j-api-parent/nd4j-api/src/main/java/org/nd4j/linalg/api/ops/impl/transforms/pairwise/arithmetic/CopyOp.java

```diff
@@ -78,7 +78,7 @@ public String onnxName() {
 
     @Override
     public String[] tensorflowNames() {
-        return new String[]{"Copy","DeepCopy","CopyHost"};
+        return new String[]{"Copy","CopyHost"};
```

Comment:
Let's also move CopyHost to Identity, like DeepCopy... CopyHost has the same single input, single output design as identity and DeepCopy